docs/pywbemcligeneraloptions.rst

```diff
@@ -644,15 +644,15 @@ the use of these operations through two general options:
 
 There are limitations with using the the ``either`` choice as follows"
 
-* The original operations did not support the filtering of responses  with a
-query language query (:ref:`--filter-query-language` and :ref:`--filter-query` )
-option which requests that the WBEM server filter
-the responses before they are returned. This can greatly reduce the size of
-the responses if effectively used but is used only when the pull operations
-are available on the server.
+* The original operations did not support the filtering of responses with a
+  query language query (``--filter-query-language`` and ``--filter-query`` )
```

Comment:
We have the capability now so how about changing the (\`\`--filter-query-language\`\` and \`\`--filter-query\`\` )  to ( :ref:\`--filter-query-language general options\` and :ref:\`--filter-query general option\`)

Reply:
We don't have that capability for these options, because we only have it for general options and these are command specific options.